typescript/sdk/src/gas/token-prices.ts

```diff
@@ -85,10 +84,11 @@ export class CoinGeckoTokenPriceGetter implements TokenPriceGetter {
 
   static withDefaultCoinGecko(
     chainMetadata: ChainMap<ChainMetadata>,
+    apiKey?: string,
     expirySeconds?: number,
     sleepMsBetweenRequests = 5000,
   ): CoinGeckoTokenPriceGetter {
-    const coinGecko = new CoinGecko();
+    const coinGecko = new CoinGeckoClient({}, apiKey);
```

Comment:
does it still behave ok if we don't provide an api key (e.g. when doing igp config in cli)

Reply:
Yes this is optional, do we use this class for doing igp config in cli?

Reply:
i'm mistaken we do not - we hit the rest api directly